### PR TITLE
Rename save_vertices.png to icon_save.png for consistency

### PR DESCRIPTION
**Pull request by merydian**

### CODE REVIEW

**Comment by merydian** [comment]
Do we need a changelog entry for this?

File: ORStools/gui/ORStoolsDialogUI.ui [hunk 1]
```diff
@@ -387,10 +387,10 @@
          <property name="checked">
           <bool>false</bool>
          </property>
-         <property name="collapsed">
```
Comment (koebi) [inline ORStools/gui/ORStoolsDialogUI.ui:390]: Do you have an explanation what these `stdset="0"` do?

Comment (merydian) [reply ORStools/gui/ORStoolsDialogUI.ui:390]: Oversaw that this changed... They appear to set the standard value for the property, with `0` meaning `False`. Should i remove them again? I think they appeared after saving in QtCreator.

Comment (koebi) [reply ORStools/gui/ORStoolsDialogUI.ui:390]: Ah. I guess so, since we don't want a value set?

Comment (merydian) [reply ORStools/gui/ORStoolsDialogUI.ui]: Done

**Review by koebi — APPROVED** [review]

**Pull request merged**